catalog/plugin/cache-plugin-set-default-metacardtags/src/main/java/org/codice/ddf/cache/plugin/metacardtags/SetDefaultMetacardTags.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.cache.plugin.metacardtags;
+
+import ddf.catalog.cache.CachePutPlugin;
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.data.types.Core;
+import java.util.Optional;
+import org.apache.commons.collections.CollectionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SetDefaultMetacardTags implements CachePutPlugin {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(SetDefaultMetacardTags.class);
+
+  private static final String ATTR = Core.METACARD_TAGS;
```

Comment:
Imo, this abstraction actually makes the code harder to follow; why not use the constant directly?

Reply:
I thought the code was cleaner. But if you feel strongly about it, I can change to use the constants directly.

Reply:
I agree that the extra step of indirection combined with a vague variable name makes it hard to understand in the local context what this plugin is doing.

Reply:
👍

Reply:
I renamed it `cache-plugin-defaultmetacardtags`